dns/py/runner.py

```diff
@@ -444,13 +439,22 @@ def _wait_for_status(self, active):
       time.sleep(1)
 
     if active:
+      break_loop = False
       while True:
-        code, out, err = self._kubectl(
+        client_pods = self._get_pod('app=dns-perf-client')
+        if client_pods == 0:
+          continue
+        for podname in client_pods:
+          code, out, err = self._kubectl(
             None,
-            'exec', _client_podname, '--',
```

Comment:
I see the issue now. This part was harcoding client podname as "dns-perf-client". Can you also remove the definition in line 34?

Reply:
Sure, and define "_client_label".